### PR TITLE
Hourly breakdown Visits and Peak

### PR DESCRIPTION
**Pull request by beweinreich**

### CODE REVIEW

File: src/actions/route-transition/explore-space-trends.ts [hunk 1]
```diff
@@ -98,7 +99,8 @@ export function calculate(space) {
   return dispatch => {
     dispatch(calculateDailyMetrics(space));
     dispatch(calculateUtilization(space));
-    dispatch(calculateHourlyBreakdown(space));
+    dispatch(calculateHourlyBreakdown(space, 'hourlyBreakdownPeaks', 'PEAKS', 'Hourly Breakdown - Avg Peak Occupancy'));
+    dispatch(calculateHourlyBreakdown(space, 'hourlyBreakdownVisits', 'Visits', 'Hourly Breakdown - Visits'));
```
Comment (1egoman) [inline src/actions/route-transition/explore-space-trends.ts]: Should this be `VISITS`?

File: src/components/explore-space-detail-hourly-breakdown-card/index.tsx [hunk 2]
```diff
@@ -18,15 +18,23 @@ export const LOADING = 'LOADING',
...
       endDate,
-      hourlyBreakdown,
+      hourlyBreakdownVisits,
+      hourlyBreakdownPeaks,
+      metric,
+      title,
     } = this.props;
 
+    let hourlyBreakdown = hourlyBreakdownVisits;
+    if(metric == "PEAKS") {
```
Comment (1egoman) [inline src/components/explore-space-detail-hourly-breakdown-card/index.tsx]: Typically you want to use `===` in javascript, not `==`.

File: src/components/explore-space-trends/index.tsx [hunk 3]
```diff
@@ -223,6 +223,17 @@ class ExploreSpaceTrends extends React.Component<any, any> {
...
+                  title="Hourly Breakdown - Visits"
+                />
+              </div>
+              <div className="explore-space-trends-item">
+                <HourlyBreakdownCard 
+                  space={space}
+                  startDate={spaces.filters.startDate}
+                  endDate={spaces.filters.endDate}
+                  metric="PEAKS"
+                  title="Hourly Breakdown - Avg Peak Occupancy"
```
Comment (1egoman) [inline src/components/explore-space-trends/index.tsx]: Might be a good idea to make the title `Hourly Breakdown - Average Peak Occupancy` (ie, `Avg` => `Average`?) Up to you.

Comment (beweinreich) [reply src/components/explore-space-trends/index.tsx]: was concerend about visibility for mobile... but we dont have mobile, so we'll tackle that in a mobile effort

**Pull request merged**